module_3/task1.js

```diff
@@ -0,0 +1,49 @@
+import EventEmitter from "./eventEmitter.js";
+
+const myEmitter = new EventEmitter();
+
+function c1() {
+    console.log('an event occurred!');
+}
+
+function c2() {
+    console.log('yet another event occurred!');
+}
+
+myEmitter.on('eventOne', c1); // Register for eventOne
+myEmitter.on('eventOne', c2); // Register for eventOne
+
+// Register eventOnce for one time execution
+myEmitter.once('eventOnce', () => console.log('eventOnce once fired'));
+myEmitter.once('init', () => console.log('init once fired'));
+
+// Register for 'status' event with parameters
+myEmitter.on('status', (code, msg)=> console.log(`Got ${code} and ${msg}`));
+
+
+myEmitter.emit('eventOne');
+
+// Emit 'eventOnce' -> After this the eventOnce will be
+// removed/unregistered automatically
+myEmitter.emit('eventOnce');
+
+
+myEmitter.emit('eventOne');
+myEmitter.emit('init');
+myEmitter.emit('init'); // Will not be fired
```

Comment:
it is not being removed after the fist emit, remember init is a once event and should be removed at the first emit, however, it is not.